lib/VCard.ts

```diff
@@ -227,8 +227,19 @@ ${name};${extended};${street};${city};${region};${zip};${country}\
     include = true,
     element: string,
   ): this {
-    const value = ''
-    this.setProperty(element, property, value)
+    let value = ''
+    let mime = ''
+
+    if (include) {
+      mime = determineMimeTypeFromUrl(url)
+      value = url || ''
+    }
+
+    this.setProperty(
+      element,
+      `${property}${mime ? `:MEDIATYPE=${mime}` : ''}`,
```

Comment:
I think this is the format for vCard 4.0 [[RFC 6350](https://tools.ietf.org/html/rfc6350#section-6.2.4)], but this library is producing output compatible with the vCard 3.0 [[RFC 2426](https://tools.ietf.org/html/rfc2426#section-3.1.4)] format only.

For URIs, I believe it should produce something like `PHOTO;VALUE=uri:http://www.abc.com/pub/photos/jqpublic.gif`.